docs/walkthrough.md

```diff
@@ -176,67 +136,85 @@ resources available in the cluster:
+  Kubernetes Name:   mariadb-10-1-26                                                                                                                                                                                                         
+  Status:            Active                                                                                                                                                                                                                  
+  Free:              true                                                                                                                                                                                                                    
+  Class:             mariadb                                                                                                                                                                                                                 
 
 Instances:
 No instances defined
 
-$ kubectl get clusterserviceplans 86064792-7ea2-467b-af93-ac9694d96d52 -o yaml
+$ kubectl get clusterserviceplans mariadb-10-1-26 -o yaml
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ClusterServicePlan
 metadata:
-  creationTimestamp: 2018-10-09T08:25:25Z
-  name: 86064792-7ea2-467b-af93-ac9694d96d52
+  creationTimestamp: "2019-06-18T03:32:31Z"
+  name: mariadb-10-1-26
   ownerReferences:
   - apiVersion: servicecatalog.k8s.io/v1beta1
     blockOwnerDeletion: false
     controller: true
     kind: ClusterServiceBroker
-    name: ups-broker
-    uid: deefbd1e-cb9c-11e8-8372-fade7e9a18e5
-  resourceVersion: "6"
-  selfLink: /apis/servicecatalog.k8s.io/v1beta1/clusterserviceplans/86064792-7ea2-467b-af93-ac9694d96d52
-  uid: df0b1a4c-cb9c-11e8-8372-fade7e9a18e5
+    name: minibroker
+    uid: 6ee99fc5-9179-11e9-8cb6-0242ac110009
+  resourceVersion: "28"
+  selfLink: /apis/servicecatalog.k8s.io/v1beta1/clusterserviceplans/mariadb-10-1-26
+  uid: b496eaf5-9179-11e9-8cb6-0242ac110009
 spec:
-  clusterServiceBrokerName: ups-broker
+  clusterServiceBrokerName: minibroker
   clusterServiceClassRef:
-    name: 4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  description: Sample plan description
-  externalID: 86064792-7ea2-467b-af93-ac9694d96d52
-  externalName: default
+    name: mariadb
+  description: Fast, reliable, scalable, and easy to use open-source relational database
+    system. MariaDB Server is intended for mission-critical, heavy-load production
+    systems as well as for embedding into mass-deployed software.
+  externalID: mariadb-10-1-26
+  externalName: 10-1-26
   free: true
 status:
   removedFromBrokerCatalog: false
 ```
 
 # Step 4 - Creating a New ServiceInstance
 
-Now that a `ClusterServiceClass` named `user-provided-service` exists within our
+Now that a `ClusterServiceClass` named `10-1-26` exists within our
```

Comment:
this should be `a 'ClusterServiceClass named 'mariadb' exists`, and all the references should be that